### PR TITLE
Sajawal/container pattern implemented

### PR DESCRIPTION
**Pull request by Sajawal-dev1**
Container  pattern Implemented

### CODE REVIEW

File: src/components/TaskFeatures.js [hunk 1]
```diff
@@ -34,11 +29,11 @@ const TaskFeatures = ({ id, title }) => {
             ></input>
             <button
               className="update-edit-btn"
-              onClick={handleSubmit((data) =>
-                dispatch(editTask({ ...data, id }), setCheck(false))
+              onClick={handleSubmit(
+                (data) => (editTask({ ...data, id }), setCheck(false))
               )}
```
Comment (IshfaqAhmedAwan) [inline src/components/TaskFeatures.js:34]: User form tag for form inputs and call handleSubmit from that form Like this :    
`<form onSubmit={handleSubmit(onSubmit)}>`
 and also use that onSubmit function to call action and set state.

File: src/components/TasksList.js [hunk 2]
```diff
@@ -1,15 +1,11 @@
...
-    return state.tasks;
-  });
+import TaskFeaturesContainer from "container/TaskFeaturesContainer";
+const TasksList = ({ todos }) => {
   return (
     <ul className="task-list">
       {todos.map((todo) => (
-        <TodoItem id={todo.id} title={todo.name} />
+        <TaskFeaturesContainer id={todo.id} title={todo.name} />
       ))}
```
Comment (ahmadfaraz629) [inline src/components/TasksList.js]: Always  give key prop when you are using **map** 
https://reactjs.org/docs/lists-and-keys.html

**Comment by ahmadfaraz629** [comment]
image

This PR has basic issues

1: Ttitle is wrong. It should be proper according to the things covered in the PR.
You could write **Container Pattern**

2: Commits should be descriptive **commit**  **issues-fixed** are not proper commits.
write a sentence of what kind of issue you resolved.
You can also check out this
https://github.com/carloscuesta/gitmoji

3: Assignees portion is missing. Assign this PR to yourself.

**Review by IshfaqAhmedAwan — APPROVED** [review]

File: src/components/TasksList.js (hunk 2 above)
Comment (Sajawal-dev1) [reply src/components/TasksList.js]: done

**Pull request merged**